src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -79,6 +79,12 @@
 
   private static final String DEFAULT_GAPIC_NAME = "gapic";
   private static final String DEFAULT_GAPIC_VERSION = "";
+  private static final String ARTIFACT_ID = "{@xsettingsClass.artifactId}";
+  private static final String GROUP_ID = "com.google.cloud";
```

Comment:
I'm not comfortable with com.google.cloud being fixed in the templates - remember, we will be generating code for APIs outside of cloud in not too long.